setup/homebrew.sh

```diff
@@ -32,15 +33,21 @@ brew install coreutils
 
 # Install GNU find, locate, updatedb, and xargs.
 # https://www.gnu.org/software/findutils/
-brew install findutils --with-default-names
+# JFV: removed --with-default-names, option deprecated
+#   Must add PATH="/usr/local/opt/findutils/libexec/gnubin:$PATH"
```

Comment:
Careful. You've made an assumption of the file location which could change from computer or computer. Instead of `/usr/local` I would suggest executing `brew --prefix`.

Also, this is a very prescriptive path. I believe homebrew creates a central place with symlinks to all executables. So you can simply add something like: PATH=$(brew --prefix)/bin:$(brew --prefix)/sbin:$PATH

I didn't test the thing above. But something like that should work for any brew executable.